composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.6",
+        "php": ">=5.3",
```

Comment:
Unless you can find a CI environment that supports PHP 5.3.0, this should stay as 5.3.6 being the minimum tested version.

Reply:
Actually i have centos with 5.3.3. But i recommend at least 5.3.2 like composer.